db/migrate/20200404103206_add_dates_to_descriptions.rb

```diff
@@ -0,0 +1,5 @@
+class AddDatesToDescriptions < ActiveRecord::Migration[6.0]
+  def change
+    add_timestamps :descriptions, null: true #TODO better default than null?
```

Comment:
Could we use any associated timestamp for migrations, e.g. the timestamp of the exercise?